### PR TITLE
Added sout in Main, added new object in Main

### PR DESCRIPTION
**Pull request by Kdxxx777**

### CODE REVIEW

File: src/test/Main.java [hunk 1]
```diff
@@ -14,5 +15,6 @@ public static void main(String[] args) {
         OrderService orderService = new OrderService();
         orderService.placeOrderOnList(order);
         System.out.println("Hello");
+        System.out.println("commiting");
```
Comment (Kdxxx777) [inline src/test/Main.java]: commiting z duzej litery

File: src/test/Main.java [hunk 2]
```diff
@@ -14,5 +15,6 @@ public static void main(String[] args) {
...
     }
```
Comment (Kdxxx777) [inline src/test/Main.java:22]: dodaj wyswietlanie aktualnej daty

File: src/test/Main.java (hunk 1 above)
Comment (kammatysiak) [reply src/test/Main.java]: done

**Review by kammatysiak — APPROVED** [review]
checked and approved
km

**Pull request closed without merging**